platform/windows/os_windows.cpp

```diff
@@ -319,8 +319,8 @@ Vector<String> OS_Windows::get_video_adapter_driver_info() const {
 	IWbemServices *wbemServices = NULL; // to get the class
 	IEnumWbemClassObject *iter = NULL;
 	IWbemClassObject *pnpSDriverObject[1]; // contains driver name, version, etc.
-	static String driver_name;
-	static String driver_version;
+	String driver_name;
+	String driver_version;
```

Comment:
If the data doesn't change while a Godot process is running, using `static` still makes sense. But it's the `Vector<String>` which should be static:
```
static Vector<String> info;
if (!info.is_empty()) {
	return info;
}

// All this convoluted logic to get the values to put in `info`.
// This logic will only need to run once even if the code is called a thousand times.
```

Reply:
yeah, I also thought about catching it early using static. But then I thought: keeping data in memory around which is usually only called once, if at all, while the Godot process is running, feels like a waste (I know it's next to nothing, but it stacks if many funcs do this)

Should I proceed with `static` anyway?

Reply:
@akien-mga: ping